moneyed/localization.py

```diff
@@ -197,6 +197,7 @@ def format(self, money, include_symbol=True, locale=DEFAULT,
 _sign(DEFAULT, moneyed.BIF, prefix='FBu')
 _sign(DEFAULT, moneyed.BMD, prefix='BD$')
 _sign(DEFAULT, moneyed.BND, prefix='B$')
+_sign(DEFAULT, moneyed.BOB, suffix=' Bs')
```

Comment:
The difference is this is a suffix sign. Most of the other suffixes has a space like this one.
```python
_sign(DEFAULT, moneyed.DKK, suffix=' Dkr')
_sign(DEFAULT, moneyed.EUR, suffix=' €')
...
```